src/core/live/live-service.js

```diff
@@ -77,7 +77,7 @@ export class LiveService {
     return this.registerUser(user)
       .then((pubnubConfig) => {
         // const copy = extend({}, pubnubConfig);
-        const pubnubClient = new PubNub(pubnubConfig);
+        const pubnubClient = new PubNub(Object.assign({ dedupeOnSubscribe: true }, pubnubConfig));
```

Comment:
Where does the `pubnubConfig` originally come from? If it is something that we create internally, should we just add this flag in before initializing the `Pubnub` client with it?

Reply:
The `pubnubConfig` is sent from the API call that is made to register a user for live service.